pkg/util/translate/translate.go

```diff
@@ -111,6 +125,17 @@ func ResetObjectMetadata(obj metav1.Object) {
 	obj.SetManagedFields(nil)
 }
 
+func IsManaged(obj runtime.Object) bool {
+	metaAccessor, err := meta.Accessor(obj)
+	if err != nil {
+		return false
+	} else if metaAccessor.GetLabels() == nil {
+		return false
+	}
+
+	return metaAccessor.GetLabels()[MarkerLabel] == Suffix
+}
```

Comment:
We already define the `IsManaged` in the Translator interface in this package, and likely this function should be removed and translate.Default.IsManaged() should be used instead.